status-im/status-react
# Add MKR to tokens

### Summary:

Add MKR token.

Additionally, fix Dai icon (which is currently MKR's icon).

### Steps to test:
- Open Status
- Go to Manage Assets, add Dai and MKR.
- You'll see that you can now add MKR, and that both tokens have their correct icons.

status: ready
